src/api/PostList/PostList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CommentList from '../CommentList/CommentList';
+import User from '../User/User';
+
+const PostList = ({ posts }) => (
+  posts.map(post => (
+    <ul>
+      <li className="post-title">
+        {post.title}
+      </li>
+      <div>
+        {post.body}
+      </div>
+
+      <div className="user">
+        <User {...post.user} key={post.id} />
+      </div>
+
+      <div>
+        <CommentList comments={post.comments} />
+      </div>
+    </ul>
```

Comment:
Same thing here, you have `ul` element for each list item. It is incorrect. You need to re-structure this component.